pkg/resource/events-logger-config/alloy/events-logger-config.alloy.yaml.template

```
@@ -64,6 +52,34 @@ networkPolicy:
       - ports:
         - port: "12345"
           protocol: TCP
+    {{- end }}
+    {{- if and .TracingEnabled (not .IsWorkloadCluster) }}
+    # Allow direct access to tempo
+    - toEndpoints:
+      - matchLabels:
+          app.kubernetes.io/name: tempo
+          io.kubernetes.pod.namespace: tempo
+      toPorts:
+      - ports:
+        - port: "4317"
+          protocol: TCP
+        - port: "4318"
+          protocol: TCP
+    {{- end }}
+    {{- if and .TracingEnabled}}
```

This looks like an useless `and`.
```suggestion
    {{- if .TracingEnabled}}
```